src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,720 @@
+
+        # Required for global pool accounting
+        jdl.append("+AcctGroup = \"%s\"\n" % (self.acctGroup))
+        jdl.append("+AcctGroupUser = \"%s\"\n" % (self.acctGroupUser))
+        jdl.append("+AccountingGroup = \"%s.%s\"\n" % (self.acctGroup, self.acctGroupUser))
+
+        # Customized classAds for this plugin
+        jdl.append('+DESIRED_Archs = \"INTEL,X86_64\"\n')
+        jdl.append('+REQUIRES_LOCAL_DATA = True\n')
+
+        return jdl
+
+    def makeSubmit(self, jobList):
+        """
+        _makeSubmit_
+
+        For a given job/cache/spec make a JDL fragment to submit the job
+
+        """
+
+        if len(jobList) < 1:
+            # I don't know how we got here, but we did
+            logging.error("No jobs passed to plugin")
+            return None
+
+        jdl = self.initSubmit(jobList)
+
+
+        # For each script we have to do queue a separate directory, etc.
+        for job in jobList:
+            if job == {}:
+                # Then I don't know how we got here either
+                logging.error("Was passed a nonexistant job.  Ignoring")
+                continue
+            jdl.append("initialdir = %s\n" % job['cache_dir'])
+            jdl.append("transfer_input_files = %s, %s/%s, %s\n" \
+                       % (job['sandbox'], job['packageDir'],
+                          'JobPackage.pkl', self.unpacker))
+            argString = "arguments = %s %i\n" \
+                        % (os.path.basename(job['sandbox']), job['id'])
+            jdl.append(argString)
+
+            jdl.extend(self.customizePerJob(job))
+
+            # Transfer the output files
+            jdl.append("transfer_output_files = Report.%i.pkl\n" % (job["retry_count"]))
+
+            # Add priority if necessary
+            task_priority = job.get("taskPriority", self.defaultTaskPriority)
+            try:
+                task_priority = int(task_priority)
+            except:
+                logging.error("Priority for task not castable to an int")
+                logging.error("Not setting priority")
+                logging.debug("Priority: %s", task_priority)
+                task_priority = 0
+
+            prio = 0
+            if job.get('priority'):
+                try:
+                    prio = int(job['priority'])
+                except ValueError:
+                    logging.error("Priority for job %i not castable to an int\n", job['id'])
+                    logging.error("Not setting priority")
+                    logging.debug("Priority: %s", job['priority'])
+                except Exception as ex:
+                    logging.error("Got unhandled exception while setting priority for job %i\n", job['id'])
+                    logging.error(str(ex))
+                    logging.error("Not setting priority")
+
+            jdl.append("priority = %i\n" % (task_priority + prio * self.maxTaskPriority))
+
+            jdl.append("+PostJobPrio1 = -%d\n" % len(job.get('potentialSites', [])))
```

Comment:
Check with unified to see what the current scheme is for this.  I'm pretty sure Unified fixes up this later to simplify negotiation.  Might as well do the correct thing on submit now.